packages/ckeditor5-ui/src/bindings/clickoutsidehandler.js

```diff
@@ -25,13 +25,14 @@
  * @param {Function} options.callback An action executed by the handler.
  */
 export default function clickOutsideHandler( { emitter, activator, callback, contextElements } ) {
-	emitter.listenTo( document, 'mousedown', ( evt, { target } ) => {
+	emitter.listenTo( document, 'mousedown', ( evt, domEvt ) => {
```

Comment:
btw, another way to write this is `{target, composedPath}`, then check `composedPath` for `undefined` later.

I'm not sure how often this code is called with multiple context elements, but right now you're calling `composedPath()` for each. It'd be slightly more efficient to generate the composed path once and reuse it:

```js
const path = composedPath !== undefined ? composedPath() : [];
for ( const contextElement of contextElements ) { ...
```

Completely optional though, since this is happening on user-interaction, we have a wide performance budget to work with - the user won't notice an extra microsecond :)

Reply:
Nevermind. This was a bad suggestion, you need to call `composedPath` with a receiver, like you already are.